api/src/main/resources/application.properties

```diff
@@ -23,6 +23,12 @@ spring.jpa.show-sql=false
 spring.jpa.hibernate.ddl-auto=validate
 server.error.include-message=always
 
+## Keep the log levels under control
+spring.main.banner-mode=off
+spring.jpa.open-in-view=false
+logging.level.root=warn
+logging.level.com.ford.labs.retroquest=info
+
```

Comment:
What is your intent behind reducing the log messages here? In our running deployments of RetroQuest, our application log traffic is fairly minimal, whether from third party frameworks or our own code. If the intent is to lower the log chattiness when running locally or when running tests, I'd prefer to set these properties within the specific properties files for those scenarios rather than make `warn` the default for the built application.  

Additionally, I prefer to lower the log level of specific third party libraries that might be too chatty, rather than lowering the default log level and then raising it for areas of concern.

Reply:
Intent is to reduce chattiness everywhere. Log traffic in the deployed environments may be minimal, but i want it to be "the minimum" :) i.e. Only log when there is something we actually care about.

I guess i just have the reverse philosophy from your statement: For all 3rd party stuff, we don't really care what it's doing unless there's a warning. For our stuff specifically, we care about infos+.

Reply:
I'm fine with merging it in the way it is, I don't think there's much practical difference either way with how RetroQuest is right now. I was just curious around where there could potentially be more direct improvements on making information more readily available to people working on code or maintenance.

Reply:
The one change I would request is to leave the ASCII banner in, Issue #294 requires it to be present.